lib/target/aquarius/classic/aqplusram.asm

```diff
@@ -65,7 +67,10 @@ setup_far_heap:
     push    af                  ;Save it
     srl     a                   ;/4
     srl     a
-    ld      de,+((((CLIB_FARHEAP_FIRST - AQPLUS_FIRST_BANK) >> 2) +1) % 256)
+    ;
+    ; Bank number is e:hl[15:14], offset is hl[13:0]
+    ;
+    ld      de,CLIB_FARHEAP_FIRST >> 2
```

Comment:
If hl is forced to 0x0 it means that the starting far heap page number must be a multiple of 4.
Adding the line below means this restriction goes away.

```
    ld    hl, CLIB_FARHEAP_FIRST <<14
```
I don't mind either way, I just wanted to point it out.

Reply:
As noted above, multiple of 4 is simpler.